django/forms/widgets.py

```diff
@@ -49,10 +49,8 @@ def __init__(self, media=None, css=None, js=None):
                 css = {}
             if js is None:
                 js = []
-
-        # create copies
-        self._css = dict(css)
```

Comment:
```python
def test_merge_copy(self):
    init_css = {'screen': ['a', 'b']}
    m = Media(css=init_css)
    m2 = m['css']
    del m2._css['screen']
    self.assertEqual(m._css, init_css)
```
To be fair, I know it didn't work before ether. I just figured it is saver to copy. I figured since the `_css` attribute is private. It should not shared across multiple instances of `Media`.